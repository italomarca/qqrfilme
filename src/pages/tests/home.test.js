import React from 'react'
import { create } from 'react-test-renderer'

import Home from '../home'

jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
  useLocation: () => ({
    pathname: '/',
  }),
}))

// todo: mock movieDbService

describe('home page', () => {
  const getComponent = () => <Home />

  it('renders correctly', () => {
    const component = create(getComponent())
    expect(component.toJSON()).toMatchSnapshot()
  })
})
