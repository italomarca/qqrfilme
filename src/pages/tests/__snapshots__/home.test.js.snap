// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page renders correctly 1`] = `
<div
  style={
    Object {
      "backgroundPosition": "center center",
      "backgroundSize": "cover",
      "display": "flex",
      "flex": 1,
      "minHeight": "100%",
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "space-between",
        "display": "flex",
        "flex": 1,
        "flexDirection": "row",
        "height": 80,
        "justifyContent": "space-between",
        "left": 0,
        "position": "fixed",
        "top": 0,
      }
    }
  >
    <div
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "flex-start",
        }
      }
    />
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "color": "#fff",
        "flex": 1,
        "fontSize": 144,
        "justifyContent": "center",
      }
    }
  >
    <p>
      PeraÃª...
    </p>
  </div>
</div>
`;
